### PR TITLE
Use clock & tick instead of waituntil

### PR DESCRIPTION
**Pull request by Tbaut**
Small refactor to prevent using `waitUntil` when we know there's a setTimeout.

### CODE REVIEW

**Comment by Tbaut** [comment]
Need to check again, somehow the clock/tick doesn't seem to work well here.

File: packages/ui/cypress/utils/clickOnConnect.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { landingPage } from '../support/page-objects/landingPage'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+
+export const clickOnConnect = () => {
+  cy.clock()
+  topMenuItems.connectButton().click()
+  landingPage.accountsLoader().should('contain', 'Loading accounts')
+  cy.tick(500)
```
Comment (Lykhoyda) [inline packages/ui/cypress/utils/clickOnConnect.ts:8]: As I understood from the documentation, it's a method to mock the time. I didn't find any setInterval in our implementation. Isn't it just pending Promise, or am I mistaken?

Comment (Tbaut) [reply packages/ui/cypress/utils/clickOnConnect.ts:8]: Nope there's a settimeout to go around a bug https://github.com/ChainSafe/Multix/blob/2a1f23bf2b9e5803a0fe7e3795e9c0f1c5579ed1/packages/ui/src/contexts/AccountsContext.tsx#L135-L138

**Comment by Tbaut** [comment]
I'm giving up. The time i spent isn't worth it imho opened https://github.com/ChainSafe/Multix/pull/403 to do a small refactor and that's it

**Pull request closed without merging**